src/services/netMgr.js

```diff
@@ -35,32 +44,36 @@ var NetMgr = {
  * @returns {boolean}
  */
 NetMgr.isAuth = function() {
-  if (this.token) {
-      var expiryTime = this.token.requestTime + this.token.expires_in;
-      if (expiryTime > Math.floor(Date.now()/1000)) {
-          return true;
-      }
-  }
-  return false;
+    if (this.token) {
+        var expiryTime = this.token.requestTime + this.token.expires_in;
+        if (expiryTime > Math.floor(Date.now()/1000)) {
+            return true;
+        }
+    }
+    return false;
 };
 
+
 /**
  * Sets the token data with real data, and preps transmission headers
  *
  * @param tokenData
  */
 NetMgr.setToken = function (tokenData) {
     this.token = tokenData;
+
     if (this.token) {
         this.token.requestTime = Math.floor(Date.now() / 1000);
         this.axiosInstance.defaults.headers.common['Authorization'] = 'Bearer ' + this.token.access_token;
+
```

Comment:
I quite like the space here as those two are variable sets and the last one is a method call. Don't mind either way - your call.